ui/components/connections/index.js

```diff
@@ -116,7 +129,13 @@ const styles = (theme) => ({
     },
   },
   expandedRows : {
-    background : `${theme.palette.secondary.default}10`,
+    background : `${theme.palette.secondary.default}10`
+  },
+  contentContainer : {
+    [theme.breakpoints.down(1050)] : {
```

Comment:
@senali-d is the the same approach that we're using in Cloud? If not, can we shift?